ghu_web/ghu_global/static/ghu_global/css/style.css

```diff
@@ -44,18 +44,24 @@ nav li {
 }
 
 nav a {
-    color: #555;
+    color: #668cff;
+    background-color: #e6ecff;
     display: inline-block;
     text-align: center;
+    font-size: 20px;
     padding: 8px;
-    border-bottom: 4px solid #aaa;
+    border-bottom: 4px solid #668cff;
 }
 
 nav a:hover {
     color: white;
-    background-color: #aaa;
```

Comment:
Combined with the `color: white` above, removing this makes navbar buttons nearly unreadable on hover:

![screenshot from 2017-05-21 12-08-18](https://cloud.githubusercontent.com/assets/431756/26285457/39661254-3e1e-11e7-8c86-4f6ad04ff937.png)

Reply:
![scrshot](https://user-images.githubusercontent.com/14301149/29977532-abcb46b4-8f0b-11e7-9c6a-15b3a7deb126.jpeg)

Reply:
That should be better.